pydash/pydash_app/user/verification_code.py

```diff
@@ -0,0 +1,15 @@
+import uuid
+from datetime import datetime, timezone, timedelta
+
+
+class VerificationCode:
+    """
+    A 'smart' randomly generated verification code that keeps track of whether it has expired.
+    Default expiration time is 7 days.
```

Comment:
This is also something to discuss with the client

Reply:
As discussed, expiration time will be changed to 24 hours/1 day. :+1: